### ISSUE TITLE
rounded class does not display on chrome

### ISSUE THREAD

**Issue by geniusit** [adoxography/tailwind-scrollbar#35]
Hi,

I try to use the scrollbar-thumb-rounded-full class with my React Next application.

However it doesn't render rounded on chrome as expected. As you can see : 

![image](https://user-images.githubusercontent.com/3659037/152868944-2e0d2ad8-fa1d-4763-ba78-985d7db3d8a6.png)

Here is my code : 

```ts
<div className="relative z-10 w-64 bg-white h-full overflow-y-auto scrollbar-thin scrollbar-thumb-gray-500 scrollbar-thumb-rounded-full">
            <Menu>
              <div className="flex flex-col">
                <div className="flex px-8 ">
                  <Menu.Button className="pl-2 my-2 w-10 rounded-full active:bg-gray-200 focus:outline-none">
                    <div onClick={() => setIsOpen(false)}>
                      <Hamburger />
                    </div>
                  </Menu.Button>
                  <Menu.Button className="px-8 focus:outline-none">
                    <Logo />
                  </Menu.Button>
                </div>
                <div className=" py-4 font-normal text-gray-500 flex-1">
                  <Menu.Items className="pl-2 pr-4 divide-y focus:outline-none" static>
                    {props.items.map(item => {

                      return (
                        <div key={item.title}>
                          <div className="pt-4 px-6 text-xs text-gray-400 uppercase">{item.title}</div>
                          <Menu.Item>
                            <MyLink href={item.href} className="rounded font-medium inline-flex items-center px-6 py-4 transition duration-200 w-full hover:bg-gray-200" onClick={() => setIsOpen(false)}>
                            {React.createElement(item.icon, {
                              /** A Technique to style current page item */
                              className: `scale-150 ${router.pathname === item.href ? "text-gray-500" : "text-gray-500"}`,
                            })}
                              <span className="px-4">{item.text}</span>
                            </MyLink>
                          </Menu.Item>
                        </div>
                      )
                    }
                    )}
                  </Menu.Items>
                </div>
              </div>
            </Menu>
          </div>
```

Why I don't see the rounded effect ? 

Thank you

**Comment by DeoThemes** [comment]
Same here, added **scrollbar: ["dark", "rounded"]** in my config, but can't get rounded classes.

**Comment by danieltejano** [comment]
same result

**Comment by Sp33ktrE** [comment]
Same, can't get it to work on Opera.

**Comment by clintnetwork** [comment]
any news @adoxography ?

**Comment by lgd8981289** [comment]
I also encountered the same problem. The query found that tailwind-scrollbar does not currently support the v3 version, but the following code can be used as a temporary solution

```
::-webkit-scrollbar {
  width: 10px;
}
::-webkit-scrollbar-track {
  background: #d9d9d9;
  border-radius: 10px;
}
::-webkit-scrollbar-thumb {
  background: #a0a0a0;
  border-radius: 10px;
}
```

**Comment by coffee377** [comment]
The reason for the problem is that the 3.0+ API has changed

###  v2.x
![t2](https://user-images.githubusercontent.com/13707072/155696691-d02e7c1d-daf3-4be0-aedb-e06bf50117e3.jpg)

### v3.x
![t3](https://user-images.githubusercontent.com/13707072/155696754-8f8e1872-8f3f-4771-8dd1-d8f17df59b90.jpg)

### **If you need it urgently. You can modify like below.** 

![3](https://user-images.githubusercontent.com/13707072/155698097-309cfa87-aad4-40fe-97d0-085313b195c2.jpg)

**Comment by leonnicklas** [comment]
**As a workaround:**

Install this [PR](https://github.com/adoxography/tailwind-scrollbar/pull/36) (by @JCayabyab) via npm until it get's merged officially...

````sh
npm install adoxography/tailwind-scrollbar#pull/36/head
````

**Comment by shishirraven** [comment]
work around by [leonnicklas](https://github.com/leonnicklas) works great! thanks![bitmoji](https://sdk.bitmoji.com/render/panel/e6518353-71e9-41b7-9208-fde8f676bf08-e4c71b61-b398-4628-a002-4aad7a0b5a44-v1.png?transparent=1&palette=1&width=246)

**Comment by adoxography** [comment]
Fix is merged into `1.3.2`, but, while revisiting it, it seems to me like `rounded` never should have been a "variant" in the first place since it's not (e.g.) `rounded:scrollbar-thumb`. Future releases will have an opt-in `nocompatible` flag to unlock this class, since my main hesitation with including it at all was its lack of a Firefox equivalent.
